ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/ModelSerDeSer.java

```diff
@@ -89,8 +89,7 @@ public static Object deserialize(byte[] modelBin) {
             // Validate the model class type to avoid deserialization attack.
             validatingObjectInputStream
                     .accept(ACCEPT_CLASS_PATTERNS)
-                    .reject(REJECT_CLASS_PATTERNS)
-                    .setObjectInputFilter(ObjectInputFilter.Config.createFilter("maxdepth=20;maxrefs=5000;maxbytes=10000000;maxarray=100000"));
```

Comment:
I remember you had more changes in your previous pull request, don't you need to revert all of them?